ParkingZoneApp/ViewModels/ParkingZones/DetailsVM.cs

```diff
@@ -0,0 +1,23 @@
+﻿using ParkingZoneApp.Models;
+
+namespace ParkingZoneApp.ViewModels.ParkingZones
+{
+    public class DetailsVM
+    {
+        public Guid Id { get; set; }
+        public string? Name { get; set; }
+        public string? Address { get; set; }
+        public DateOnly? CreatedDate { get; set; }
```

Comment:
```suggestion
        public string Name { get; set; }
        public string Address { get; set; }
```

Reply:
If I remove CreatedDate how can call it from details view?
![image](https://github.com/dilmurod-rozikov/ParkingZone/assets/141717735/22b303aa-2899-4f55-9b57-4bce5fcf75fb)